etna/transforms/missing_values/resample.py

```diff
@@ -140,26 +140,30 @@ def __init__(
         self.in_column = in_column
         self.distribution_column = distribution_column
         self.inplace = inplace
-        self.out_column = self._get_out_column(out_column)
+        self.out_column = out_column
```

Comment:
Why didn't out tests fails because of the creation warning?

Reply:
Currently, we don't have translation of warnings into errors (we commented it in `pyproject.toml`). Eaerlier we probably added smth to ignore this warning as a temporary fix.